src/main/scala/encry/view/EncryNodeViewHolder.scala

```diff
@@ -51,13 +51,29 @@ class EncryNodeViewHolder[StateType <: EncryState[StateType]] extends Actor with
     ADProofs.modifierTypeId -> ADProofSerializer,
     Transaction.ModifierTypeId -> TransactionSerializer
   )
+  var txsInBlocks: Int = 0
 
   system.scheduler.schedule(5.second, 5.second) {
     if (settings.influxDB.isDefined)
       system.actorSelection("user/statsSender") !
         HeightStatistics(nodeView.history.bestHeaderHeight, nodeView.history.bestBlockHeight)
   }
 
+  system.scheduler.schedule(5 second, 1 second) {
```

Comment:
replace `system` with `context`, and schedule message sending only if influxDb is defined